app/controllers/tmp/iterations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Tmp::IterationsController < ApplicationController
```

Comment:
We can use controllers in this directory for development purposes. They can have zero test coverage and will be thrown away before we launch. But for things like the button I've added below they are very useful for quickly testing things.